pkg/combustion/templates/13-embedded-registry.sh.tpl

```diff
@@ -4,7 +4,7 @@ set -euo pipefail
 mkdir /opt/hauler
 
 mount /usr/local
-mv {{ .EmbeddedRegistryTar }} /opt/hauler/{{ .EmbeddedRegistryTar }}
+mv ./registry/* /opt/hauler/
```

Comment:
I'd suggest to template the directory value. As mentioned in one of the previous PRs, it's very easy for the Go implementation to start using something different than a `registry` directory which will completely invalidate the script but the tests would continue passing.

Reply:
Sounds good 👍